CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Set sentry-native backend to `crashpad` by default and `breakpad` for Windows ARM64 ([#2791](https://github.com/getsentry/sentry-dart/pull/2791))
```

Comment:
The code below doesn't actually change to breakpad for windows (note: also the PR title is wrong). In order to do that, we'd need to check in cmake. I'm also fine with leaving the sentry-native default. 

```suggestion
- Set sentry-native backend to `crashpad` by default ([#2791](https://github.com/getsentry/sentry-dart/pull/2791))
```

related: https://github.com/getsentry/sentry-native/issues/880

Reply:
```
if(FLUTTER_TARGET_PLATFORM EQUAL "windows-arm64")
    set(native_backend "breakpad")
else()
    set(native_backend "crashpad")
endif()
```

this is set in our windows cmake, so I think that's used if I'm not mistaken

Reply:
looks good then